ghpc.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Was this change intentional? It appears this line is 4 spaces in other files, it's better to keep it consistent.

Reply:
Not intentional. Change was proposed by the linter, and I just went with it. 
I reverted it for now.